t/graphics/xcolors.xml

```diff
@@ -332,307 +332,307 @@ A=100*13 = 1300pt;</p>
           </tr>
           <tr>
             <td align="left" thead="row"><text font="sansserif" fontsize="90%">red</text></td>
-            <td align="left"><text backgroundcolor="#FF0000" class="ltx_phantom" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> 1 0 0</text></td>
-            <td align="left"><text backgroundcolor="#FF0000" class="ltx_phantom" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> 0 1 1 0</text></td>
-            <td align="left"><text backgroundcolor="#FF0000" class="ltx_phantom" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> 0 1 1</text></td>
-            <td align="left"><text backgroundcolor="#FF0000" class="ltx_phantom" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> FF0000</text></td>
-            <td align="left"><text backgroundcolor="#4D4D4D" class="ltx_phantom" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> 0.3</text></td>
+            <td align="left"><text backgroundcolor="#FF0000" class="ltx_phantom" cssstyle="padding:0.0pt" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> 1 0 0</text></td>
```

Comment:
Quickly marking that the test using `ltx_phantom` text now gets explicit 0 padding. 

Probably intentional?

Reply:
I think so.